ideascube/conf/idb_fra_vdp2.py

```diff
@@ -0,0 +1,4 @@
+"""Ideaxbox for Ville de Paris, France"""
+from .idb_fra import *  # noqa
+
+IDEASCUBE_NAME = u"Ville de Paris"
```

Comment:
You can remove `IDEASCUBE_NAME`, given it's a new server and you have it at hand.